### PR TITLE
Grid field action menu item

### PR DESCRIPTION
**Pull request by ivoba**
Added option in constructor to add copy icon either as column or to the Actions pulldown.

### CODE REVIEW

File: code/GridField/CopyButton.php [hunk 1]
```diff
@@ -29,40 +38,59 @@ public function augmentColumns($gridField, &$columns)
...
 
+    private function getCopyAction($gridField, $record, $columnName)
+    {
+        $title = _t('GridAction.Copy', 'Copy');
+        $field = GridField_FormAction::create(
+            $gridField,
+            'CopyRecord'.$record->ID,
+            false,
+            "copyrecord",
+            ['RecordID' => $record->ID])
```
Comment (dhensby) [inline code/GridField/CopyButton.php]: this is some really weird code formatting. Can you place the `)` on the next line inline with the `$field` declaration and then move the subsequent arrow calls to be one tab from the `)`?

**Review by dhensby — CHANGES_REQUESTED** [review]
Minor code style change, please :)

File: code/GridField/CopyButton.php [hunk 2]
```diff
@@ -81,12 +81,13 @@ private function getCopyAction($gridField, $record, $columnName)
...
             "copyrecord",
-            ['RecordID' => $record->ID])
-                                     ->addExtraClass('gridfield-button-copy')
-                                     ->setAttribute('classNames', 'font-icon-plus')
-                                     ->setAttribute('title', $title)
-                                     ->setDescription(_t('GridAction.COPY_DESCRIPTION', 'Copy'))
-                                     ->setAttribute('aria-label', $title);
+            ['RecordID' => $record->ID]
+        )
+        ->addExtraClass('gridfield-button-copy')
```
Comment (dhensby) [inline code/GridField/CopyButton.php]: Ah, sorry, I meant these calls to be one more tab in.

Can you make that change then squash your commits 😇 

thanks

Comment (ivoba) [reply code/GridField/CopyButton.php]: Like this?
Ill squash when approved.

**Review by dhensby — APPROVED** [review]

**Pull request merged**